Documentation/network/bgp-control-plane.rst

```diff
@@ -304,18 +304,23 @@ By configuring ``authSecretRef`` for a neighbor you can configure that a
 peer.
 
 ``authSecretRef`` should reference the name of a secret in the BGP secrets
-namespace (if using the Helm chart this is ``cilium-bgp-secrets`` by default).
-The secret should contain a key with a name of ``password``.
+namespace (if using the Helm chart this is ``kube-system`` by default). The
+secret should contain a key with a name of ``password``.
 
 BGP secrets are limited to a configured namespace to keep the permissions
-needed on each Cilium Agent instance to a minimum. The Helm chart will create
-this namespace and configure Cilium to be able to read from it by default.
+needed on each Cilium Agent instance to a minimum. The Helm chart will
+configure Cilium to be able to read from it by default.
 
 An example of creating a secret is:
 
 .. code-block:: shell-session
 
-   $ kubectl create secret generic -n cilium-bgp-secrets --type=string secretname --from-literal=password=my-secret-password
+   $ kubectl create secret generic -n kube-system --type=string secretname --from-literal=password=my-secret-password
+
+If you wish to change the namespace, you can set the
+``bgpControlPlane.secretNamespace.name`` Helm chart value. If you wish to
+create the namespace, you can set the
+``bgpControlPlane.secretNamespace.create`` to ``true``.
```

Comment:
Can we re-word this second sentence a bit? Helm creates the namespace, not the user, so it doesn't seem right to phrase it as "if the user wishes to create the namespace". Here's a suggestion:

```suggestion
``bgpControlPlane.secretNamespace.name`` Helm chart value. To have the
namespace created automatically, you can set the 
``bgpControlPlane.secretNamespace.create`` Helm chart value to ``true``.
```

Reply:
Thanks @learnitall - I have addressed this (as Yutaro is on vacation ATM)